files/build_templates/init_cfg.json.j2

```diff
@@ -16,5 +16,21 @@
             "{{crm_res}}_high_threshold": "85"{% if not loop.last %},{% endif -%}
 {% endfor %}
         }
+    },
+    "FEATURE": {
+{%- for feature in ["sflow", "telemetry"] %}
+        "{{feature}}": {
+            "status": "disabled"
+        }{% if not loop.last %},{% endif -%}
+{% endfor %}
+    },
+    "CONTAINER_FEATURE": {
+{%- for container in ["bgp", "database", "dhcp-relay", "lldp", "pmon", "radv", "sflow",
+                      "snmp", "swss", "syncd", "teamd", "telemetry"] %}
```

Comment:
Can you also add `restapi `to this list?

Reply:
It looks like the actual container name is `rest-api`:

https://github.com/Azure/sonic-buildimage/blob/ebb07d99424de7421baffa10b9cb2ee91ac61494/rules/docker-restapi.mk#L19

Reply:
Added.